### PR TITLE
fix/TR-2853/item-media-resolver-update

### PR DESCRIPTION
**Pull request by augustas**
Just a small fix to propagate inline source resolution to parent resolver. 
 
Related to : https://oat-sa.atlassian.net/browse/TR-2853
  
  
#### How to test
 
Described in accompanying PR https://github.com/oat-sa/extension-tao-itemqti/pull/1961
 
  
#### Dependencies
 
Requires :
 - [ ] https://github.com/oat-sa/tao-core/pull/3252
   
Companion PR :
 - [ ] https://github.com/oat-sa/extension-tao-itemqti/pull/1961

### CODE REVIEW

File: models/classes/media/ItemMediaResolver.php [hunk 1]
```diff
@@ -41,7 +41,7 @@ public function __construct($item, $lang)
     public function resolve($url)
     {
         $urlParts = parse_url($url);
-        if (!isset($urlParts['scheme']) || !isset($urlParts['host'])) {
+        if (!isset($urlParts['scheme']) || (!isset($urlParts['host']) && $urlParts['scheme'] !== 'data')) {
```
Comment (boajer) [inline models/classes/media/ItemMediaResolver.php:44]: Year field in Copyright block should be updated too

**Review by emgolubev — APPROVED** [review]
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

File: models/classes/media/ItemMediaResolver.php (hunk 1 above)
Comment (augustas) [reply models/classes/media/ItemMediaResolver.php:44]: Right, I've updated it 👍🏻

**Review by wazelin — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Pull request merged**